extensions/vscode/src/constants.ts

```diff
@@ -74,6 +74,7 @@ const homeViewCommands = {
   NavigateToDeploymentContent:
     "posit.publisher.homeView.navigateToDeployment.Content",
   ShowContentLogs: "posit.publisher.homeView.navigateToDeployment.ContentLog",
+  EditCurrentConfiguration: "posit.publisher.homeView.edit.Configuration",
```

Comment:
It looks like the command added was `posit.publisher.edit.Configuration` and did not include the `homeView`. We can change it here or in the `package.json`